shap/explainers/_tree.py

```diff
@@ -132,28 +152,6 @@ def __init__(
         if getattr(self.masker, "clustering", None) is not None:
             raise ExplainerError("TreeExplainer does not support clustered data inputs! Please use shap.Explainer or pass an unclustered masker!")
 
-        # check for deprecated options
```

Comment:
don't we need this part of the code? At least the model_output parameter?

Reply:
Which part is still needed? As I understand, the deleted lines just handled deprecated aliases for the function arguments. The `model_output` parameter is still defined as a keyword argument to the function, so that variable still exists

Reply:
You're right, looks fine to remove these lines.